R/drive_mime_type.R

```diff
@@ -105,3 +105,12 @@ one_ext <- function(type) {
   }
   m
 }
+
+drive_show_mime_type <- function(file) {
+  file <- as_dribble(file)
+  file <- confirm_some_files(file)
+  file[["mime_type"]] <- NULL
```

Comment:
Aha! Sounds like something I should address in `promote()`, instead of using this workaround everywhere, though. Agree?